lms/templates/learner_dashboard/certificate.underscore

```diff
@@ -0,0 +1,7 @@
+<div class="certificate-container">
+    <p class="title"><%- gettext('XSeries Program Certificates') %>:</p>
+    <img src="/static/images/xseries-certificate-visual.png" alt="XSeries Program Certificates" aria-hidden="true">
```

Comment:
Also, should the image url be hardcoded into the template, or would it make sense to pass the url from python to the model to make updating it easier? I'm not sure of the level of effort vs benefits here, @jimabramson @rlucioni @schenedx is there a precedent for this sort of thing already in edx-platform?